pvlib/location.py

```diff
@@ -356,3 +359,92 @@ def get_sun_rise_set_transit(self, times, method='pyephem', **kwargs):
                              'one of pyephem, spa, geometric'
                              .format(method))
         return result
+
+
+def lookup_altitude(latitude, longitude, filepath=None):
+    """
+    Look up location altitude from local low-resolution altitude map
+    supplied with pvlib. The data for this map comes from multiple open data
+    sources with varying resolutions aggregated by Mapzen.
+
+    More details can be found here
+    https://github.com/tilezen/joerd/blob/master/docs/data-sources.md
+
+    Altitudes from this map are a coarse approximation and can have
+    significant errors (100+ meters) introduced by downsampling and
+    source data resolution.
+
+    Parameters
+    ----------
+    latitude : float.
+        Positive is north of the equator.
+        Use decimal degrees notation.
+
+    longitude : float.
+        Positive is east of the prime meridian.
+        Use decimal degrees notation.
+
+    filepath : None or string, default None
+        The path to the ``.h5`` file.
```

Comment:
If we keep the `filepath` kwarg (it feels like a good idea), we should provide a description of the .h5 file content, so that a user could substitute their own.

Reply:
I'm leaning towards removing the file as an input. 

The code here is nothing special just look-up code.

Given that the look-up code is not very flexible right now, I think most people would rather look up the altitudes in their native encoding and not have to re-encode them to our specs.
Maybe there is a case for having a more flexible decoding code and a global override for the altitude map, but I don't think it is worth it; passing the altitude to Location or clear_sky is simple enough.

I will remove it for now, but if there is any interest in a more comprehensive solution, I can modify this PR or make a new one afterward.